src/services/anchor/index.ts

```diff
@@ -150,7 +150,10 @@ export const sep10 = async (
   const { signingKey, webAuthEndpoint } = tomlValues;
 
   if (!exists(signingKey) || !exists(webAuthEndpoint)) {
-    throw new Error('Missing values in TOML file');
+    throw new Error('sep10: Missing values in TOML file');
+  }
+  if (!address) {
+    throw new Error('sep10: Address must be defined at this stage');
   }
```

Comment:
Maybe we should just change the function signature to not allow passing an undefined address 😅

Reply:
Moved to the caller of `sep10` now, but we can't really get rid of the condition there unless we want to touch address in `useVortexAccount`. I think it's fine though.